sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -2133,12 +2133,6 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase with Compilat
         "inputTypesLen" -> bound.inputTypes().length.toString))
   }
 
-  def commandUnsupportedInV2TableError(name: String): Throwable = {
-    new AnalysisException(
-      errorClass = "_LEGACY_ERROR_TEMP_1200",
-      messageParameters = Map("name" -> name))
-  }
```

Comment:
We can remove the error class too.

Reply:
To reduce the complexity of this PR, it will be placed in a separate PR to complete the renaming or removing of the error class-related operations.

Reply:
In fact, just remove the error class, it will not increase the complexity.

Reply:
Yes, I misunderstood your meaning. That's right, we can remove it from `error-classes. json`.